src/plugins/recordTypes/intake/forms/default.jsx

```diff
@@ -0,0 +1,155 @@
+const template = (configContext) => {
+  const {
+    React,
+  } = configContext.lib;
+
+  const {
+    Col,
+    Cols,
+    Panel,
+    Row,
+  } = configContext.layoutComponents;
+
+  const {
+    Field,
+    InputTable,
+  } = configContext.recordComponents;
+
+  return (
+    <Field name="document">
+      <Panel name="objectEntryInfo" collapsible>
```

Comment:
These fields need to be rearranged, because the tab order no longer makes sense. I'll take care of it after this gets merged.